app/views/conference/schedule.html.erb

```diff
@@ -25,3 +25,19 @@
       <%= render 'all_events' %>
   <% end %>
 </div>
+
+<%= javascript_tag do %>
```

Comment:
okey, but I think we should merge #1030 before, to avoid a lot of merge conflicts.